ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/EthPeer.java

```diff
@@ -130,6 +133,15 @@ public ResponseStream send(final MessageData messageData) throws PeerNotConnecte
     }
   }
 
+  public void propagateBlock(final Block block, final UInt256 totalDifficulty) {
+    final NewBlockMessage newBlockMessage = NewBlockMessage.create(block, totalDifficulty);
+    final Capability capability = Capability.create("eth", 63);
```

Comment:
We don't want to create a new capability here and need to use the passed in `protocolName`.  I suspect `connection.sendForProtocol(protocolName, newBlockMessage)` is what you want.